src/TheThingsNetwork.cpp

```diff
@@ -251,17 +251,19 @@ int TheThingsNetwork::sendBytes(const byte* buffer, int length, int port, bool c
   str.concat(port);
   if (!sendCommand(str, buffer, length)) {
     debugPrintLn(F("Send command failed"));
-    return 0;
+    return -1;
   }
 
   String response = readLine(10000);
   if (response == "") {
     debugPrintLn(F("Time-out"));
```

Comment:
Not sure if we should always print this debug line or only when we asked for a confirm